index.ts

```diff
@@ -182,8 +182,12 @@ const partitionGenAthenaAccess = new aws.iam.RolePolicyAttachment("partition-ath
     policyArn: aws.iam.ManagedPolicies.AmazonAthenaFullAccess
 });
 
+// const cron = new aws.cloudwatch.EventRule("hourly-cron", {
+//     scheduleExpression: "rate(1 hour)"
+// });
+
 const cron = new aws.cloudwatch.EventRule("hourly-cron", {
-    scheduleExpression: "rate(1 hour)"
+    scheduleExpression: "rate(1 minute)"
```

Comment:
nit: can we clean this up? I think a better solution here might be to use an entry in the `pulumi.*.yaml` file. Something like `dev: true`

Then we can do:

```ts
const isDev = config.get("dev");
const cronUnit = isDev ? "minute" : "hour";
const scheduleExpression  = `rate(1 ${cronUnit)`
```

Reply:
Completed, but may want to add for event creator at a later point